test/rekt/resources/broker/broker.go

```diff
@@ -58,7 +58,7 @@ func init() {
 }
 
 func WithEnvConfig() []manifest.CfgFn {
-	cfg := []manifest.CfgFn{WithBrokerClass(EnvCfg.BrokerClass)}
+	cfg := []manifest.CfgFn{WithBrokerClass("MTChannelBasedBroker")}
```

Comment:
this should not be necessary, you should be able to `export BROKER_CLASS=MTChannelBasedBroker` before running the tests to force MTChannelBasedBroker if necessary.